app/src/main/java/eu/kanade/tachiyomi/ui/manga/chapter/ChaptersSortBottomSheet.kt

```diff
@@ -148,6 +153,35 @@ class ChaptersSortBottomSheet(controller: MangaDetailsController) :
             binding.chapterFilterLayout.setAsDefaultFilter.isInvisible = true
             binding.chapterFilterLayout.resetAsDefaultFilter.isInvisible = true
         }
+        binding.filterGroupsButton.isVisible = presenter.allChapterScanlators.isNotEmpty()
+
+        binding.filterGroupsButton.setOnClickListener {
+            val scanlators = presenter.allChapterScanlators.toList()
+            val filteredScanlators =
+                presenter.manga.filtered_scanlators?.let { ChapterUtil.getScanlators(it) }
+                    ?: scanlators.toSet()
+            val preselected = if (filteredScanlators.isNullOrEmpty()) {
+                scanlators.mapIndexed { index, _ -> index }
+            } else {
+                filteredScanlators.map { scanlators.indexOf(it) }
+            }.toIntArray()
+
+            MaterialDialog(activity!!)
+                .title(R.string.filter_groups_title)
+                .listItemsMultiChoice(
+                    items = scanlators,
+                    initialSelection = preselected,
+                    allowEmptySelection = false
+                ) { _, selections, _ ->
+                    val selected = selections.map { scanlators[it] }.toSet()
+                    presenter.setScanlatorFilter(selected)
+                }
+                .negativeButton(R.string.reset_group_filter) {
```

Comment:
For the sake of tachiyomi translators, lets just use "reset" here and remove reset_group_filter. When i get to changing the checkboxes into x's i'll probably have this clear all the checked  and not close the dialog yet